README.md

```diff
@@ -293,6 +277,36 @@ Publishes a new helm chart package (`.tgz`) to a helm chart repository
           token: ${{ secrets.BOT_GITHUB_TOKEN}}
 ```
 
+### send-slack-notif
+
+Sends a slack notification with a pre-defined payload, relying on the [slackapi/slack-github-action](https://github.com/slackapi/slack-github-action) official action.
+
+```yaml
+      - uses: Alfresco/alfresco-build-tools/.github/actions/send-slack-notif@AAE-9076-setup-gh-slack-notifs
+        with:
+          channel-id: 'channel-id'
+          token: ${{ secrets.SLACK_BOT_TOKEN }}
+          notif-color: '#A30200'
+```
+
+### rtCamp/action-slack-notif
+
+[rtCamp/action-slack-notify](https://github.com/rtCamp/action-slack-notify) can be used to send a slack notification,
+here is a sample:
+
+```yml
+      - name: Slack Notification
+        if: ${{ failure() && steps.is_default_branch.outputs.result == 'true' }}
+        uses: rtCamp/action-slack-notify@v2
+        env:
+          SLACK_COLOR: ${{ job.status }}
+          SLACK_WEBHOOK: ${{ secrets.APA_SLACK_WEBHOOK }}
+```
+
+This requires [setting up a slack webhook](https://slack.com/help/articles/115005265063-Incoming-webhooks-for-Slack?eu_nc=1) for the target slack channel.
+
+
+
```

Comment:
at this point I prefer to remove this external action and rely on ours - if we are not providing reasons for choosing this one instead of our new custom one

Reply:
I agree: I copy/pasted it and forgot to remove it, will do :+1: